.ldrelease/publish-circleci.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Run this in publish step after all version information have been updated.
+set -ev
+
+brew install circleci
```

Comment:
I don't think this should be in the script. We can run a `which` to verify the user has it installed and error out if it isn't there

Reply:
We need the `circleci` cli installed in the releaser environment. If we don't install it, we won't be able to automate any of the circleci steps. I could be missing something obvious here, but I don't think installing the circleci cli is an issue?

Reply:
we should install it in the container if it's not there currently, `brew` is pretty mac dependent, while the releaser containers are using linux.

Reply:
looks like it's based on Ubuntu. So we can change it use `apt-get` and I think it should be good to go. Thinking on it more yeah this script is really only used in the release process.

Reply:
if we go this route though I think we'll need to install with curl to bash which isn't ideal https://circleci.com/docs/2.0/local-cli/#alternative-installation-method

Reply:
Good point re `brew` being osx specific. I've changed this now to use apt to install curl and then installing circleci using curl.